.github/workflows/image-scan.yml

```diff
@@ -1,26 +1,26 @@
-name: Image Scan
+name: Docker Image Scan
 
-on:
-  pull_request:
-    branches:
-      - master
+on: push
+  # push:
+  #   branches:
+  #     - master
+  #     - '*security*'
```

Comment:
Why is this commented out? Is this a step we used to do but aren't doing any more?

Reply:
That shouldn't look like that. Thanks for pointing it out.